src/server/middleware/sendHtml.js

```diff
@@ -115,7 +115,7 @@ export function renderModuleScripts({
     const { integrity, url: src } = moduleMap.modules[moduleName][bundle];
     const { key } = moduleMap;
     const additionalAttributes = isDevelopmentEnv ? '' : `integrity="${integrity}"`;
-    const scriptSource = isDevelopmentEnv ? src : `${src}?key=${key}`;
+    const scriptSource = isDevelopmentEnv || !key ? src : `${src}?key=${key}`;
```

Comment:
are we naming it `key`? 

@PixnBits do your thing ...

Reply:
This is a discussion between @PixnBits and @10xLaCroixDrinker